### ISSUE TITLE
Uncaught exception in browser when there's a #modules element on the page

### ISSUE THREAD

**Issue by oleggromov** [dfilatov/inherit#27]
Hi there!

I have a weird issue with inherit:
- there's a `#modules` div on the page
- the browser creates a global object `window.modules` representing this `Element` (I cannot guess if any spec covers this behavior)
- browser throws an exception trying to evaluate:

```js
    if(typeof modules === 'object') {
	    modules.define('inherit', function(provide) {
	        provide(inherit);
	    });
	    defineAsGlobal = false;
	}
```

Maybe it's better to test `modules` not only to be an object, but to have a right class too?

**Comment by dfilatov** [comment]
@oleggromov Good catch, I'll fix it asap

**Comment by oleggromov** [comment]
https://github.com/dfilatov/inherit/pull/28

**Comment by dfilatov** [comment]
fixed in 2.2.4

**Comment by oleggromov** [comment]
Thanks! 
👍
